### PR TITLE
[test](mv)  Add mv case from fe ut

### PR DESCRIPTION
**Pull request by BBB-source**
# Proposed changes
add some mv case from fe ut MaterializedViewFunctionTest

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/materialized_view_p0/testAggQueryOnAggMV3/testAggQueryOnAggMV3.groovy [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import org.codehaus.groovy.runtime.IOGroovyMethods
+
+suite ("testAggQueryOnAggMV2") {
```
Comment (BiteTheDDDDt) [inline regression-test/suites/materialized_view_p0/testAggQueryOnAggMV3/testAggQueryOnAggMV3.groovy:20]: testAggQueryOnAggMV3

File: regression-test/suites/materialized_view_p0/testAggQueryOnAggMV2/testAggQueryOnAggMV2.groovy [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import org.codehaus.groovy.runtime.IOGroovyMethods
+
+suite ("testAggQueryOnAggMV2") {
+    sql """ DROP TABLE IF EXISTS emps; """
```
Comment (Yukang-Lian) [inline regression-test/suites/materialized_view_p0/testAggQueryOnAggMV2/testAggQueryOnAggMV2.groovy:21]: table name needs to be modified, the regression test is parallel, and the same table name will cause an error.

File: regression-test/suites/materialized_view_p0/testAggQueryOnAggMV3/testAggQueryOnAggMV3.groovy [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    sql """ DROP TABLE IF EXISTS emps; """
```
Comment (Yukang-Lian) [inline regression-test/suites/materialized_view_p0/testAggQueryOnAggMV3/testAggQueryOnAggMV3.groovy:21]: ditto

File: regression-test/suites/materialized_view_p0/testAggQueryOnAggMV2/testAggQueryOnAggMV2.groovy (hunk 2 above)
Comment (BiteTheDDDDt) [reply regression-test/suites/materialized_view_p0/testAggQueryOnAggMV2/testAggQueryOnAggMV2.groovy:21]: > table name needs to be modified, the regression test is parallel, and the same table name will cause an error.

Those cases were placed in different directory and running on different databases, so it is ok.

Comment (Yukang-Lian) [reply regression-test/suites/materialized_view_p0/testAggQueryOnAggMV2/testAggQueryOnAggMV2.groovy:21]: Sorry I didn't notice it, thanks for your remind. 😃

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0 Regression

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run build all

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run buildall

**Comment by BBB-source** [comment]
run P0

**Comment by BBB-source** [comment]
run P0 Regression (ARM pipeline)

**Comment by BBB-source** [comment]
run p0

**Comment by BiteTheDDDDt** [comment]
run p0

**Comment by BiteTheDDDDt** [comment]
run arm

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Pull request merged**